### PR TITLE
New english words, translated to Catalan & Spanish

### PR DESCRIPTION
**Pull request by marc2332**
## Changes
- Added vampire, witcher and zombie to English
- Translated to Spanish
- Translated to Catalan

### CODE REVIEW

**Review by LucasCtrl — APPROVED** [review]

File: lang/cat.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+module.exports = {
+  lang: 'cat', //Catalan
```
Comment (LucasCtrl) [inline lang/cat.js]: Referring to this list the Catalan use the `ca` code.

**Review by LucasCtrl — CHANGES_REQUESTED** [review]
You also need to change the file name referring to this doc http://www.mathguide.de/info/tools/languagecode.html

**Comment by marc2332** [comment]
Should be fine now 👍🏻

**Review by LucasCtrl — APPROVED** [review]

**Comment by LucasCtrl** [comment]
The bot will be updated during the night with the new translation file 😉

**Pull request merged**